src/python/pants/python/python_setup.py

```diff
@@ -131,9 +131,15 @@ def register_options(cls, register):
         register(
             "--resolver-jobs",
             type=int,
-            default=None,
+            default=2,
```

Comment:
I wasn't sure if 2 or 1 is a better default.

I'm pretty sure that the CPU count is a poor default because that would mean that by default, `total parallelism == cpu count ^ 2`, which could be way too high.